src/figuresClasses.ts

```diff
@@ -1,19 +1,73 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape = 'triangle';
 
-}
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('The digit cannot be less than 0 or equal to 0');
```

Comment:
You repeat text of this error in each class so move it to the variable

Reply:
Ok, thanks for the advice :)